### PR TITLE
LPS-155728, LPS-140258, LPS-158992, LPS-160177, LPS-160240 | master

### PR DESCRIPTION
**Pull request by gaborlovasdotcom**
[LPS-155728](https://issues.liferay.com/browse/LPS-155728)
[LPS-140258](https://issues.liferay.com/browse/LPS-140258)
[LPS-158992](https://issues.liferay.com/browse/LPS-158992)
[LPS-160177](https://issues.liferay.com/browse/LPS-160177)
[LPS-160240](https://issues.liferay.com/browse/LPS-160240)

Developer comments:

- Plain text emailAddress and userLogin in AuditLogContext & AuditMessage
- Removed sessionId from AuditLogContext

cc @stian-sigvartsen , @JingyuanZhao

### CODE REVIEW

**Review by stian-sigvartsen — APPROVED** [review]
Looks good to me. I can see we are just reducing the scope of the existing test to not expect the `sessionId`. We are not testing the `userLogin` field but I think that is fine for now

**Comment by stian-sigvartsen** [comment]
Sorry, I should have been clearer in the comment. I can see we are asserting that the `userLogin` field exists, but not that its value matches the user login used to start the authenticated session. i.e. email address, screen name or user id. And I don't think we don't need to go that far right now 👍

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/Audit.testcase [hunk 1]
```diff
@@ -71,6 +71,31 @@ definition {
...
+
+			PortletEntry.save();
+
+			User.logoutPG();
+		}
+
+		task ("Assert that sessionId is not present in the audit log context") {
+			Navigator.openURL(baseURL = "http://localhost:8080/api/jsonws/user/get-user-by-email-address/company-id/email-address");
+
+			AssertConsoleTextNotPresent(value1 = "AuditLogContext.sessionId");
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/Audit.testcase:95]: @JingyuanZhao , could you please add extra assertation for the context and the event title as written in the AC, or do we already have unit/integration tests covering that? @stian-sigvartsen

**Review by gaborlovasdotcom — COMMENTED** [review]
See my comment

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/Audit.testcase [hunk 2]
```diff
@@ -71,6 +71,31 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-140258. Assert that sessionId is not present in the audit log context."
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/Audit.testcase:74]: Please mention the TC-1 as well

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/Audit.testcase [hunk 3]
```diff
@@ -71,6 +71,31 @@ definition {
...
+	@priority = "2"
+	test AssertSessionIdIsNotPresentInAuditLogContext {
+		property portal.acceptance = "ture";
+		property test.name.skip.portal.instance = "Audit#AssertSessionIdIsNotPresentInAuditLogContext";
+
+		task ("Enable log context and Incoming X-Request-ID") {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/Audit.testcase:80]: Given-When-Then format pls

**Comment by gaborlovasdotcom** [comment]
ci:test:stable

**Comment by gaborlovasdotcom** [comment]
ci:test:relevant

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:bundle

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Comment by gaborlovasdotcom** [comment]
Sending directly in https://github.com/brianchandotcom/liferay-portal/pull/123425

**Comment by gaborlovasdotcom** [comment]
Waiting for https://issues.liferay.com/browse/LPS-161370 to be ready before forwarding to Brian.

**Comment by gaborlovasdotcom** [comment]
ci:test:bundle

**Comment by oliverkecskemety** [comment]
Added [LPS-161370](https://issues.liferay.com/browse/LPS-161370) in #858

**Pull request closed without merging**